Client/Extensions/UIAlertControllerExtensions.swift

```diff
@@ -187,4 +195,106 @@ extension UIAlertController {
 
         return deleteAlert
     }
-}
+
+    /**
+     Builds the Alert view that asks if the users wants to add a third party search engine.
+
+     - parameter okayCallback: Okay option handler.
+
+     - returns: UIAlertController for asking the user to add a search engine
+     **/
+
+    class func addThirdPartySearchEngine(okayCallback: (UIAlertAction) -> Void) -> UIAlertController {
```

Comment:
Hm, I'm not sure `UIAlertControllerExtensions` is the best place to add these. Generally, I think it's considered good practice to add extensions with generic methods that can be used anywhere, but you'd extending `UIViewController` for a very specific use case. I think it'd be better if these method were moved into some third-party-search-specific class instead. 

(I see we did this above with `deleteLoginAlertWithDeleteCallback`, but I think that was also a mistake).